testsuite/pytests/test_iaf_wang_2002.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+#
+# test_iaf_wang_2002.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Tests synaptic dynamics of iaf_wang_2002. Since the neuron is conductance based,
```

Comment:
Put brief description on first line separated by blank line. Applies to all docstrings.

Make even more explicit that you test here against the approximation made, not the full model.

And one could in principle set up a test for models with untractable dynamics by using a SciPy ODE solver.